apps/codeforafrica/src/components/OurTeam/OurTeam.js

```diff
@@ -0,0 +1,40 @@
+import { Section } from "@commons-ui/core";
+import { Button, Grid } from "@mui/material";
+import Box from "@mui/material/Box";
+import Image from "next/image";
+import React from "react";
+
+import TeamContent from "./TeamContent";
+
+import regionIcon from "@/codeforafrica/assets/images/Africa@2400x 1.png";
+
+const OurTeam = React.forwardRef(function OurTeam(props, ref) {
+  return (
+    <Box ref={ref} sx={{ bgcolor: "primary.light" }}>
+      <Section
+        sx={{
+          px: { xs: 2.5, sm: 0 },
+          py: { xs: 5, md: 8, lg: 10 },
```

Comment:
These don't seem to match designs: xs: 40px (5 is correct), sm: 30px, md: 62px... eveything above `md` seem to use `md` py.